config/manifest/deployment.yaml

```diff
@@ -127,6 +130,10 @@ spec:
             timeoutSeconds: 5
             failureThreshold: 4
             successThreshold: 1
+          # Failing to provide a writable $HOME can cause TUF client initialization to panic
```

Comment:
The comment does not match the volume location. See comment below on volume location.